Dockerfile

```diff
@@ -24,8 +24,4 @@ RUN poetry config virtualenvs.create false && \
 
 COPY . /app/
 
-RUN useradd -r -u 1000 runner-manager \
```

Comment:
what happened?

Reply:
I'm not exactly sure, but I believe the issue stems from the fact that port 80 is only accessible by the `root` user. Therefore, I will try to change the port from 80 to 8080 everywhere